### PR TITLE
User Guide - Run `cog` on Windows 11 using WSL 2

### PR DESCRIPTION
**Pull request by afiaka87**
Finally got this written up - was more painful than I was expecting, but considering the references you have to dig through to figure this all out normally, hopefully it saves folks some time.

### CODE REVIEW

**Comment by afiaka87** [comment]
@andreasjansson @zeke @bfirsh @evilstreak may be interested.

**Review by andreasjansson — APPROVED** [review]
This is brilliant, thanks for going through this arduous process and writing it up! Can we link this doc from https://github.com/replicate/cog#next-steps as well?

File: docs/wsl2/wsl2.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+- [4. Configure WSL 2](#4-configure-wsl-2)
+- [5. Configure CUDA WSL-Ubuntu Toolkit](#5-configure-cuda-wsl-ubuntu-toolkit)
+- [6. Install Docker](#6-install-docker)
+- [7. Install `cog` and pull an image](#7-install-cog-and-pull-an-image)
+- [8. Run a model in WSL 2](#8-run-a-model-in-wsl-2)
+- [9. References](#9-references)
+
+Running cog on Windows is now possible thanks to WSL 2. Follow this guide to enable WSL 2 and GPU passthrough on Windows 11. 
+
+Windows 10 is not officially supported, as you need to be on an insider build in order to use GPU passthrough.
```
Comment (zeke) [inline docs/wsl2/wsl2.md:15]: Maybe not necessary, but do you think it's worth having a section for prerequisites at the start here, so people know what they'll need before they get too deep?

**Review by zeke — COMMENTED** [review]
This is looking good! I left a few suggestions.

File: docs/wsl2/wsl2.md (hunk 1 above)
Comment (afiaka87) [reply docs/wsl2/wsl2.md]: Yeah, definitely. I added a section to the top explaining the requirement for Win11 and that you need an NVIDIA GPU.

**Comment by zeke** [comment]
@allcontributors please add @afiaka87 for docs

**Pull request merged**